.golangci.yaml

```diff
@@ -0,0 +1,92 @@
+# todo: this should match the configuration of osv-scanner for the most part
+#   currently it does not because using that config raises type errors,
+#   in addition to some of the rules having violations that need addressing,
+#   so the current focus is on getting as many linters as possible running
+
+output:
+  sort-results: true
+linters:
+  # todo: currently linting raises a type error, so until that's resolved we've got to be very selective in what we enable
+  enable:
```

Comment:
Is there a linter rule that checks for unneeded type declarations such as the ones removed in https://github.com/google/osv-scalibr/pull/261 ? If so let's disable that for now as we currently can't guarantee that internally submitted code will adhere to that.

Reply:
that's a feature of `gofmt` (specifically, my understanding is that's the `-s` flag which is "simplify") - so we'll want to hold off on landing #264 for now

Reply:
I see. Yeah if that's introduced in #264 then it should be fine to submit this and hold off merging #264.